Documentation/workflow/NativeDebuggingUsingAndroidStudio.md

```diff
@@ -0,0 +1,21 @@
+# Native Debugging of Mono using Android Studio
+
+With these steps it is possible to start a XA app with managed debugging in Visual Studio (on Mac and Windows) and native debugging of mono (with debugging symbols) using Android Studio.  
+
+Steps:
+1: Get the same github mono version that is in Android SDK
```

Comment:
Overall: Please use normal Markdown, so that it renders "nicely."

In particular, this should be `1.`, not `1:`.

Reply:
I don't understand what this line *means*.  The *Android* SDK doesn't contain Mono; I assume this should say the `Xamarin.Android SDK`?

Furthermore, it's left unsaid how to determine the "same GitHub mono version".  I assume on macOS this could be done by clicking **Visual Studio** > **About Visual Studio**, then clicking the **Show Details** button, then looking for the **Xamarin.Android** > **Build Information** section, a'la:

```
Mono: mono/mono/2018-08-rc@5ad371dab1b
Java.Interop: xamarin/java.interop/d16-0@c987483
LibZipSharp: grendello/LibZipSharp/master@44de300
LibZip: nih-at/libzip/rel-1-5-1@b95cf3f
MXE: xamarin/mxe/xamarin@b9cbb535
ProGuard: xamarin/proguard/master@905836d
SQLite: xamarin/sqlite/3.26.0@325e91a
Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-0@0a7edd6
```

For example, this would imply using mono/mono/2018-08-rc@5ad371dab1b.

Next, how do you get it?  Presumably:

```bash
cd external/mono
git checkout 5ad371dab1b
```